commons/src/main/java/com/powsybl/commons/json/JsonUtil.java

```diff
@@ -273,4 +274,24 @@ public static void skip(JsonParser parser) throws IOException {
             }
         } while (objectCount != 0);
     }
+
+    public static void assertLessThanOrEqualToReferenceVersion(String tag, String version, String referenceVersion) {
+        Objects.requireNonNull(version);
+        if (version.compareTo(referenceVersion) > 0) {
+            String exception = String.format(
+                "LoadflowParameters. Tag: %s is not a valid property for LoadflowParameters version %s. LoadFlowParameters version should be <= %s %n",
+                tag, version, referenceVersion);
+            throw new PowsyblException(exception);
+        }
+    }
+
+    public static void assertGreaterThanReferenceVersion(String tag, String version, String referenceVersion) {
+        Objects.requireNonNull(version);
+        if (version.compareTo(referenceVersion) <= 0) {
+            String exception = String.format(
+                "LoadflowParameters. Tag: %s is not a valid property for LoadflowParameters version %s. LoadFlowParameters version should be > %s %n",
+                tag, version, referenceVersion);
+            throw new PowsyblException(exception);
+        }
+    }
```

Comment:
The error message are too specific. Make these functions more generic (pass the message as a supplier) or move these methods in the LoadFlow API?

Reply:
Done